### PR TITLE
Added display titles for pages

### PR DESCRIPTION
**Pull request by rahulkurumkar**
Added title to pages.
MissingTitles

### CODE REVIEW

File: src/app/plugins/kubernetes/components/namespaces/NamespacesListPage.js [hunk 1]
```diff
@@ -7,32 +7,43 @@ import { createUsePrefParamsHook } from 'core/hooks/useParams'
...
-      />
-    </PageContainer>
+    const [namespaces, loading, reload] = useDataLoader(
+      namespaceActions.list,
+      params
+    )
+    return (
+      <PageContainer floatingHeader={false}>
+        <Tabs>
+          <Tab value="namespace" label="Name Spaces">
```
Comment (xantorres) [inline src/app/plugins/kubernetes/components/namespaces/NamespacesListPage.js]: Namespaces should be a single word, not two

File: src/app/plugins/kubernetes/components/namespaces/NamespacesListPage.js [hunk 2]
```diff
@@ -42,14 +53,14 @@ export const options = {
   columns: [
     { id: 'name', label: 'Name' },
     { id: 'clusterName', label: 'Cluster' },
-    { id: 'created', label: 'Created' },
+    { id: 'created', label: 'Created' }
```
Comment (xantorres) [inline src/app/plugins/kubernetes/components/namespaces/NamespacesListPage.js]: we are adding trailing commas (eslint `comma-dangle`) to our multilined objects and arrays, as it makes adding new items to the object/array better when checking for differences/rebasing.

Comment (rahulkurumkar) [reply src/app/plugins/kubernetes/components/namespaces/NamespacesListPage.js]: Fixed it.

**Comment by rahulkurumkar** [comment]
> Approved but definitely address the minor stuff Xan brought up before merging.

Addressed the review comments.

File: src/app/plugins/kubernetes/components/namespaces/NamespacesListPage.js (hunk 1 above)
Comment (rahulkurumkar) [reply src/app/plugins/kubernetes/components/namespaces/NamespacesListPage.js]: Changed it

**Pull request merged**